clients/cobol-lsp-vscode-extension/src/services/snippetcompletion/cobolSnippets.json

```
@@ -1380,19 +1422,97 @@
   },
   "IF": {
     "prefix": "IF",
-    "body": ["IF ${1:CONDITION}", "   $0", "END-IF"],
+    "body": ["IF ${1:CONDITION}", "\t${2:ifTrue}", "END-IF"],
```

I think the idea was to exit the snippet and start writing the statements.
```suggestion
    "body": ["IF ${1:CONDITION}", "\t$0", "END-IF"],
```

---

Won't change as the current way keeps it in line with the `IF ELSE` snippet. If the user tabs twice, they'll immediately start replacing the `ifTrue` text.